src/content_mapblock.h

```diff
@@ -92,6 +92,7 @@ class MapblockMeshGenerator
 
 // liquid-specific
 	bool top_is_same_liquid;
+	bool draw_bottom;
```

Comment:
Irrelevant for regular nodes -> `draw_liquid_bottom`

Reply:
Renamed.